sdks/python/src/opik/evaluation/metrics/llm_judges/hallucination/metric.py

```diff
@@ -12,6 +13,11 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class HallucinationResponseFormat(BaseModel):
+    verdict: str
```

Comment:
we can make the type better since we expect the verdict to always be one of 2 words.
```python
verdict: Literal["hallucinated", "factual"]  # string values can probably be imported but you've got the point
```
same comment is relevant for all other response formats in this PR.

Reply:
I'm going to update the template to return a score directly, it's odd to ask the LLM to return a string and then convert that to a number.